models/Room.js

```diff
@@ -0,0 +1,27 @@
+import mongoose from 'mongoose';
+
+const RoomSchema = mongoose.Schema(
+  {
+    name: {
+      type: String,
+      required: true,
+      trim: true,
+    },
+    description: {
+      type: String,
+      required: true,
+      trim: true,
+    },
+    // friends: [
+    //   {
+    //     type: mongoose.Schema.Types.ObjectId,
+    //     ref: 'Friends',
+    //   },
```

Comment:
u might want an `id` field in the schema

Reply:
@eden-chan I think by this, Jimmy means you need to store to uuid of the space (when you create a space notice that the url is /room/<unique_id>. This uuid is how we differentiate our spaces. Storing just the name and description is not enough.